integrations/database/mariadb/kubearchive.sql

```diff
@@ -59,6 +59,7 @@ CREATE TABLE `log_url` (
   `uuid` uuid NOT NULL,
   `url` text NOT NULL,
   `container_name` text NOT NULL,
+  `json_path` text NOT NULL,
```

Comment:
This should not be `NOT NULL`. It's possible that some logging system will not need it.

@maruiz93 , take note.

Reply:
done

Reply:
How about just `DEFAULT ''` rather than `NOT NULL`.  I guess it really doesn't matter too much.

But to me `NOT NULL` implies I'd better pass it a real JSON path.  And that shouldn't be required.

Reply:
And what does "done" mean here?  You didn't change anything.

Reply:
I removed `NOT NULL` from the jsonPath line. I can add `DEFAULT ''` to be safe though

Reply:
No need. Sorry, I always look at the old code so I didn't see your change.